Hints.py

```diff
@@ -511,6 +511,16 @@ def get_barren_hint(spoiler, world, checked):
 
 def is_not_checked(location, checked):
     return not (location.name in checked or get_hint_area(location)[0] in checked)
+    
+def is_not_checked_dual(world, dual, checked):
```

Comment:
Instead of adding a new function, I recommend modifying `is_not_checked` to accept either a single location or list of locations like I suggested for `add_hint`.